toolkit/locales/en-US/chrome/global/aboutSupport.ftl

```diff
@@ -45,6 +65,13 @@ aboutSupport-appBasicsMultiProcessSupport = Multiprocess Windows
 aboutSupport-appBasicsSafeMode = Safe Mode
 aboutSupport-appBasicsProfiles = Profiles
 
+# This string displays the number of open multiprocess (e10s) windows compared
+# to the overall number of open windows, and the current status of e10s in the
+# browser.
+# $remote (Number) - Number of multiprocess windows
+# $total (Number)  - Total number of open windows
+# $status (String) - Current status of e10s
```

Comment:
If the localizer writes:

    { $remote }/{ $total } ({ $status })

and `$status` is 0, then the result will be:

    1/3 (0)

That's because `$status` is a number and it's inserted as such into the string.

Instead, we want to use `$status` to select one of the text variant given after the `->` operator:

```
 { $remote }/{ $total } ({ $status ->
   [0] Enabled by user
  *[other] Unknown status
 })
```

Now the result will be:

    1/3 (Enabled by user)